src/ulog.c

```diff
@@ -159,11 +159,15 @@ static void print_color_end(log_target *tgt, ulog_Event *ev) {
 
 static void print_time_sec(log_target *tgt, ulog_Event *ev) {
 
+    char buf[10];
```

Comment:
Let's make it event better and replace magic numbers with TIME_STAMP_BUF_SIZE and FULL_TIME_STAMP_BUF_SIZE